### PR TITLE
insertShiftArray challenge

### PR DESCRIPTION
**Pull request by Ginsusamurai**

### CODE REVIEW

**Comment by Ginsusamurai** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: code-challenges/401/arrayShift/array-shift.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  for(let i = 0; i < arr.length; i++){
+    if(i === trigger){
+      tempArr.push(val);
+    }
+    tempArr.push(arr[i]);
+  }
+  return tempArr;
+}
+
```
Comment (adrianhuebner) [inline code-challenges/401/arrayShift/array-shift.js:36]: It would be a great exercise to come back to this and write it without using any of the built in methods, that would look like:
- Declare a variable that is a midpoint
- Loop through the array until you hit the midpoint
- Using if conditionals reassign the value of i
